Tests/FunctionalTests/RunFunctionalTests.js

```diff
@@ -85,11 +86,13 @@ function cleanUpDocker() {
     run("docker stop ainjsmysql");
     run("docker stop ainjsredis");
     run("docker stop ainjspostgres");
+    // run("docker stop ainjsmssql");
```

Comment:
Are these commented on purpose?

Reply:
Yes it seems like mssql is not compatible with github runners. I don't want to block this PR on it though